python/ray/tests/test_metrics_head.py

```diff
@@ -20,7 +21,7 @@ def test_metrics_folder():
             f"{session_dir}/metrics/grafana/provisioning/dashboards/default.yml"
         )
         assert os.path.exists(
-            f"{session_dir}/metrics/grafana/provisioning/dashboards"
```

Comment:
why do we change this btw?

Reply:
It was a weird pattern for the dashboard files and the grafana provisioning files to be in the same place. It makes more sense to have these be separate folders to avoid config files conflicting with each other.